runbotics/runbotics-scheduler/src/database/guest/guest.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository, UpdateResult } from 'typeorm';
+import { GuestEntity } from './guest.entity';
+import { Guest } from 'runbotics-common';
+
+@Injectable()
+export class GuestService {
+    constructor(
+        @InjectRepository(GuestEntity)
+        private guestRepository: Repository<GuestEntity>,
+    ) { }
+
+    async findById(userId: number): Promise<Guest> {
+        return this.guestRepository.findOne({ where: { userId }});
+    }
```

Comment:
include in the name that you mean userId

Reply:
done